tests/migration_test.go

```diff
@@ -848,6 +848,9 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				By("Checking that the VirtualMachineInstance console has expected output")
 				Expect(console.LoginToFedora(vmi)).To(Succeed())
 
+				By("Stressing the VMI")
+				runStressTest(vmi, stressdefaultVMSize)
+
```

Comment:
@vladikr do we need to kill the stress test after we finish attempting parallel migrations in order to guarantee the test will successfully migrate?

Reply:
@vladikr ping.

Reply:
We've discussed this offline and I've suggested letting `runStressTest` to accept timeout as a parameter and use a short timeout for stress in this test.

Reply:
Added in latest force-push.